freezeyt_blog/articles/meeting34.md

```diff
@@ -0,0 +1,30 @@
+# Freezeyt sraz třicátý čtvrtý - adaptér na Frozen-Flask
+
+Začali jsme review PR. První PR specifikoval verzi Pytestu
+v requirements, protože MonkeyPatch, který používáme,
+je součástí Pytestu od verze 6.2.0., takže těm,
+kdo mají nainstalovanou starší verzi Pytestu padají testy s chybou.
+
+Druhý PR se zabývá reorganizací testů.
+Řešili jsme, co se mění.
+David by mohl napsat tuto část 😉
```

Comment:
@nappex Could you write a sentence or two about your PR? If you think, that this is enough, then just delete the last sentence.

Reply:
> @nappex Could you write a sentence or two about your PR? If you think, that this is enough, then just delete the last sentence.

Ok I take a look on that